__tests__/app.test.js

```diff
@@ -101,22 +102,21 @@ describe("GET /api/articles", () => {
       .get("/api/articles")
       .expect(200)
       .then((res) => {
-        res.body.result.forEach((article, index) => {
-          if (index < res.body.result.length - 1) {
-            expect(
-              article.created_at > res.body.result[index + 1].created_at
-            ).toBe(true);
-          }
-        });
+        expect(res.body.result).toBeSorted({ descending: true });
```

Comment:
We want to be specific here which is descending (we want this to be created_at - have a look at the jest sorted docs and how to give another argument of which bit you want to sorted